index.js

```diff
@@ -137,9 +137,16 @@ class SafeView extends Component {
     );
   }
 
-  _onLayout = (...args) => {
+  _onLayout = (info, ...args) => {
     if (!this.view) return;
+    if (!info || !info.nativeEvent) {
+      if (this.props.onLayout) {
+        this.props.onLayout(...args);
```

Comment:
we need to pass the `info` var through here. i'd call it `e` or `event` instead, also. to see why it's necessary to pass it through have a look at https://snack.expo.io/@notbrent/thrilled-pizza - run it and then look at logs, you'll see this:

<img width="902" alt="screen shot 2018-09-19 at 12 26 43 pm" src="https://user-images.githubusercontent.com/90494/45776297-5adc9a80-bc07-11e8-9934-b266c292b542.png">